doc/api_queries.rst

```diff
@@ -110,6 +114,11 @@
 
     Evaluate the query. Returns a list of matching :class:`dnf.package.Package` instances.
 
+  .. method:: unneeded(sack, yumdb, debug_solver=False)
```

Comment:
Jarda, please, make it private now. Once we have swdb and proper API to it we can expose this too.